recipes-bsp/tegra-binaries/tegra-libraries-multimedia-v4l_32.6.1.bb

```diff
@@ -0,0 +1,22 @@
+DEPENDS = "tegra-libraries-core tegra-libraries-cuda tegra-libraries-multimedia tegra-libraries-multimedia-utils"
+
+require tegra-libraries-common.inc
+
+TEGRA_LIBRARIES_TO_INSTALL = "\
+    tegra/libnvcuvidv4l2.so \
+    tegra/libtegrav4l2.so \
+"
+
+do_install() {
+    install_libraries
+    install -d ${D}${libdir}/libv4l/plugins/ ${D}${libdir}/libv4l/plugins-wrapped/
+    for f in libv4l2_nvvideocodec.so libv4l2_nvcuvidvideocodec.so; do
```

Comment:
`libv4l2_nvcuvidvideocodec.so` library is not supported for `tegra210`
Maybe we have to add condition to check if library exists.
Something like:

```
 do_install() {
     install_libraries
     install -d ${D}${libdir}/libv4l/plugins/ ${D}${libdir}/libv4l/plugins-wrapped/
     for f in libv4l2_nvvideocodec.so libv4l2_nvcuvidvideocodec.so; do
          if [ -e "${DRVROOT}/tegra/$f" ]; then
               install -m 0644 ${DRVROOT}/tegra/$f ${D}${libdir}/libv4l/plugins/
          fi
     done
     install -m 0644 ${DRVROOT}/tegra/libv4l2_nvvidconv.so ${D}${libdir}/libv4l/plugins-wrapped/
 }
```